src/olympia/addons/models.py

```diff
@@ -638,6 +639,8 @@ def delete(self, msg='', reason='', send_delete_email=True):
             file_tasks.hide_disabled_files.delay(addon_id=self.id)
 
             self.versions.all().update(deleted=True)
+            VersionReviewerFlags.objects.filter(version__addon=self).update(
+                pending_rejection=None)
```

Comment:
Do we need something similar for addon disable, or is that handled a different way?  (I guess the versions get disabled at that point and... somehow the flags get removed...?)

Reply:
The cron will eventually reject them even if they are disabled, so we don't need to do anything. For deletions we explicitly clear it mostly for convenience, because it's not something that is supposed to be reverted so we know we can get rid of the flag right away.

Reply:
(Also, yes, rejecting versions would clear the flag)